src/main/java/duke/io/Ui.java

```diff
@@ -37,14 +58,28 @@ public void printError(String... error) {
         System.out.println(OUTPUT_HORIZONTAL_LINE);
     }
 
+    /**
+     * Prints a horizontal bar.
+     */
     public void printHorizontalBar() {
         System.out.println(OUTPUT_HORIZONTAL_LINE);
     }
 
+    /**
+     * Prints one or more lines of text with ASCII accents.
+     *
+     * @param lines A variable number of strings, each would print a line of text.
+     */
     public void stylizedPrint(String... lines) {
```

Comment:
Hi Sam! As both stylizedPrint methods "print" stuff, would it be better to name them with verbs (similar to your printHorizontalBar), i.e. printStylized? Cheers!

Reply:
Hi Hardy :)
`printStylized()` does sound better, good suggestion